app/modules/api/audio_event_parser.rb

```diff
@@ -319,6 +322,54 @@ def parse_raven(contents)
         .map(&:to_h)
     end
 
+    def avianz?(content)
+      # Need to differentiate from other JSON formats
+      # there's been a reviewer key in the JSON for nearly 6 years
+      # https://github.com/smarsland/AviaNZ/blame/794343335b81d5fa6a3be299930eab8ba5a139cd/Segment.py#L391
+      content&.start_with?('[') && content.include?('Reviewer')
+    end
```

Comment:
Discussed offline, seems error prone since "Reviewer" is an optional column

https://github.com/smarsland/AviaNZ/blame/794343335b81d5fa6a3be299930eab8ba5a139cd/Segment.py#L388-L391

https://github.com/search?q=repo%3Asmarsland%2FAviaNZ%20saveJSON&type=code